cmd/eskimo/users_test.go

```diff
@@ -0,0 +1,267 @@
+	"github.com/stretchr/testify/require"
+)
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUsers_Success_Common(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+	defer cancel()
+
+	var allUsers []string
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	ctx = context.WithValue(ctx, "requestingUserIDCtxValueKey", userID) //nolint:staticcheck // No way
+
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("create many users", func() {
+		allUsers = bridge.MustCreateMutuallyReferringUsers(ctx, t)
+	})
+	WHEN("get all users", func() {
+		body, status = bridge.GetUsers(ctx, t, "u", token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("verify returned users", func() {
+			verifyMutuallyReferringUsers(t, body, allUsers)
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUsers_Failure_RegExp(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	ctx = context.WithValue(ctx, "requestingUserIDCtxValueKey", userID) //nolint:staticcheck // No way
+
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("empty DB", func() {})
+	WHEN("get all users", func() {
+		body, status = bridge.GetUsers(ctx, t, "u", token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("should be empty", func() {
+			assert.Equal(t, body, "[]")
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUserByID_Success(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("create single user", func() {
+		bridge.MustCreateDefaultUser(ctx, t, userID, token, "bogus.user")
+	})
+	WHEN("get user by ID", func() {
+		body, status = bridge.GetUserByID(ctx, t, userID, token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("user received well", func() {
+			verifyUserProfile(t, body, userID, "bogus.user")
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUserByID_Failure_NoUser(t *testing.T) {
```

Comment:
```suggestion
func TestService_GetUserByID_Failure_UserNotFound(t *testing.T) {
```
better describes the test.

Reply:
done